docs/user-guide/ingest-data/for-observability/elasticsearch.md

```diff
@@ -125,3 +133,36 @@ output.elasticsearch:
 ```
 
 The `parameters` section is optional, while `hosts` and `index` should be adjusted according to your actual setup.
+
+### Telegraf
+
+If you are using [Telegraf](https://github.com/influxdata/telegraf) to collect logs, you can use its Elasticsearch plugin to write data to GreptimeDB, as shown below:
+
+```toml
+[[outputs.elasticsearch]]
+  urls = [ "http://localhost:4000/v1/elasticsearch" ]
+  index_name = "test_table"
+  health_check_interval = "0s"
+  enable_sniffer = false
+  flush_interval = "1s"
+  manage_template = false
+  template_name = "telegraf"
+  overwrite_template = false
+  namepass = ["tail"]
+
+ [outputs.elasticsearch.headers]
+    "X-GREPTIME-DB-NAME" = "public"
+    "X-GREPTIME-PIPELINE-NAME" = "greptime_identity"
+
+[[inputs.tail]]
+  files = ["/tmp/test.log"]
+  from_beginning = true
+  data_format = "value"
+  data_type = "string"
+  character_encoding = "utf-8"
+  interval = "1s"
+  pipe = false
+  watch_method = "inotify"
+```
+
+Please adjust `urls`, `index_name`, and the corresponding `outputs.elasticsearch.header` according to your actual setup.
```

Comment:
what url should greptimedb config use?

Reply:
Add more descriptions for the collector configurations.